utils/entrypoint.sh

```diff
@@ -405,13 +410,17 @@ else
 
     mysql -u root -e "CREATE USER 'zmuser'@'localhost' IDENTIFIED BY 'zmpass';"
     mysql -u root -e "GRANT ALL PRIVILEGES ON *.* TO 'zmuser'@'localhost';"
+    mysql -u root -e "ALTER USER 'zmuser'@'localhost' IDENTIFIED WITH mysql_native_password BY 'zmpass';"
```

Comment:
Why? mysql_native_password is deprecated in modern mysql.

Reply:
![image](https://github.com/ZoneMinder/zmdockerfiles/assets/13343271/2ab19bc7-9b2a-4b71-85f4-cca455013426)

Because if the use of mysql_native_password is not allowed, Mysql will fail to initialize the connection.

Reply:
Maybe you should update database connection code in zoneminder?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/caching-sha2-pluggable-authentication.html seems like the now-default auth mechanism requires either a TLS connection or RSA-based auth, neither of which are worth it at all for a connection to a DB server inside the same container. It makes sense to switch it back to `mysql_native_password` imo to keep things simple.

Reply:
mysql_native_password is DEPRECATED.  If you use it with modern mysql your logs will be spammed endlessly about it.